src/bubble_sort/Example2.java

```diff
@@ -0,0 +1,47 @@
+package bubble_sort;
+
+import java.util.Random;
+
+public class Example2 {
+    public static void main(String[] args) {
+        int[] array = new int[10];
+        Random generator = new Random();
+        for (int i = 0; i < array.length; i++) {
```

Comment:
please leave a empty line before basic procedures (for, while, if, etc.)

Reply:
Done